spec/beaker/hypervisor/vmpooler_spec.rb

```diff
@@ -147,20 +143,15 @@ module Beaker
     describe "#cleanup" do
 
       it "cleans up hosts in the pool" do
-        MockVsphereHelper.powerOn
-
+        mock_http = MockNet::HTTP.new("host", "port")
         vmpooler = Beaker::Vmpooler.new( make_hosts, make_opts )
-        allow( vmpooler ).to receive( :require ).and_return( true )
-        allow( vmpooler ).to receive( :sleep ).and_return( true )
         vmpooler.provision
-        vmpooler.cleanup
+        vm_count = vmpooler.instance_variable_get( :@hosts ).count
 
-        hosts = vmpooler.instance_variable_get( :@hosts )
-        hosts.each do | host |
-          name = host.name
-          vm = MockVsphereHelper.find_vm( name )
-          expect( vm.runtime.powerState ).to be === "poweredOn" #handed back to the pool, stays on
-        end
+        expect( Net::HTTP ).to receive( :new ).exactly(vm_count).times.and_return( mock_http )
```

Comment:
please pick a spacing style for method parameters & apply it consistently across your changes.